libraries/Microsoft.Bot.Builder.AI.QnA/Utils/CardHelper.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.Bot.Schema;
+
+namespace Microsoft.Bot.Builder.AI.QnA
+{
+    public class CardHelper
+    {
+        /// <summary>
+        /// Get Hero card.
+        /// </summary>
+        /// <param name="suggestionsList">List of suggested questions.</param>
+        /// <param name="cardTitle">Title of the cards.</param>
+        /// <param name="cardNoMatchText">No match text.</param>
+        /// <returns>IMessageActivity.</returns>
+        public static IMessageActivity GetSuggestionsCard(List<string> suggestionsList, string cardTitle, string cardNoMatchText)
```

Comment:
This method doesn't really make sense in it's own class. 

... also a "/Utils/" directory seems like an incomplete idea. There has to be a better place / name.

Reply:
We will introduce one more function for Multi-turn prompts, so we need one class for Card Builder.